uber/models.py

```diff
@@ -904,6 +912,12 @@ def hotel_status(self):
         else:
             return 'Hotel nights: ' + hr.nights_display
 
+class WatchList(MagModel):
+    first_name = Column(UnicodeText)
+    last_name = Column(UnicodeText)
+    disabled = Column(Boolean, default=False)
```

Comment:
This version of the watch list is really just what AnthroCon asked for, which mirrors their old system. We can definitely discuss implementation details when I re-write this as a generic plugin.

I think disabling watch list items rather than deleting them is good, actually - it means we have the ability to look back through old items to make sure someone's not a repeat offender. I don't know about implementing it for every table, but for here I think it makes sense.